### PR TITLE
Fix/change mapbox location picking

### PR DESCRIPTION
**Pull request by SerhiiKundys**

### CODE REVIEW

**Review by SerhiiKundys — COMMENTED** [review]
neet to remove unnecessary data

File: client/src/components/events/FindLocationForm.vue [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        this.input.setAttribute('type', 'search');
+        this.$el.appendChild(this.input);
+
+        this.placesAutocomplete = places({
+            appId: VUE_APP_ALGOLIA_APP_ID,
+            apiKey: VUE_APP_ALGOLIA_API_KEY,
+            container: this.input
+        });
+        this.placesAutocomplete.configure({
+            language: 'en'
```
Comment (mv-v-7) [inline client/src/components/events/FindLocationForm.vue:41]: Does ALGOLIA have Ukrainian localization? It is question only for future in this stage of project English autocomplete will be enough

Comment (aLLeXUs) [reply client/src/components/events/FindLocationForm.vue:41]: yeah, can we remove language to use the default one (it will use browser's default)?

Comment (mv-v-7) [reply client/src/components/events/FindLocationForm.vue:41]: You can get current language from state /store/modules/i18n/  see  getter [i18n.GET_CURRENT_LANGUAGE].
If exist autocomplete for ua using of this getter good solution. But as I mention before maybe we can stay only for english autocompletion

**Review by mv-v-7 — APPROVED** [review]
Exist problem with localization of autocomplete, but to get working prototype we can go straight.

**Pull request merged**